projects/ngrx.io/content/guide/component-store/read.md

```diff
@@ -173,6 +173,59 @@ private readonly fetchMoviesData$ = this.select(
 );
 </code-example>
 
+## `selectSignal` method
+
+ComponentStore also provides the `selectSignal` method, which has two signatures. The first signature creates a signal from the provided state projector function, while the second creates a signal by combining provided signals, similar to the select method that combines provided observables.
+
+<code-example header="users.store.ts">
+import { Injectable } from '@angular/core';
+import { ComponentStore } from '@ngrx/component-store';
+
+import { User } from './user.model';
+
+type UsersState = { users: User[]; query: string };
+
+@Injectable()
+export class UsersStore extends ComponentStore&lt;UsersState&gt; {
+  // type: Signal&lt;User[]&gt;
+  readonly users = this.selectSignal((s) => s.users);
+  // type: Signal&lt;string&gt;
+  readonly query = this.selectSignal((s) => s.query);
+  // type: Signal&lt;User[]&gt;
+  readonly filteredUsers = this.selectSignal(
+    this.users,
+    this.query,
+    (users, query) =>
+      users.filter(({ name }) => name.includes(query))
+  );
+}
+</code-example>
+
+Similar to the `computed` function, the `selectSignal` method also accepts the equality function to stop the recomputation of the deeper dependency chain if two values are determined to be equal.
```

Comment:
Do we need this here? 
In the API docs this will be documented already.

If we keep it, I think mentioning (or removing the reference) that `computed` comes from Angular is useful otherwise it could be confusing if people start looking for it in our docs.

Reply:
I think it makes sense to remove the reference, like this?
```suggestion
The `selectSignal` method also accepts an equality function to stop the recomputation of the deeper dependency chain if two values are determined to be equal.
```

A code example might also be helpful, wdyt?

Reply:
Looks good to me, thanks! 👍